src/binding/c/attr_api.txt

```diff
@@ -0,0 +1,202 @@
+# vim: set ft=c:
+
+MPI_Attr_delete:
+    .desc: Deletes an attribute value associated with a key on a
```

Comment:
key on a "communicator"? I think it is dropped when you copy-pasted.

Reply:
It gets dropped during a script extraction. My original scraping script wasn't perfect and I then manually corrected them as I find the errors. Thanks for pointing out the missing ones!

Reply:
Fixed in #4955